src/ScratchStorage.js

```diff
@@ -190,13 +190,14 @@ class ScratchStorage {
      * @param {?DataFormat} [dataFormat] - Optional: load this format instead of the AssetType's default.
      * @param {Buffer} data - Data to store for the asset
      * @param {?string} [assetId] - The ID of the asset to fetch: a project ID, MD5, etc.
+     * @param {?object} urlParams - object of params to add to querystring
      * @return {Promise.<object>} A promise for asset metadata
      */
-    store (assetType, dataFormat, data, assetId) {
+    store (assetType, dataFormat, data, assetId, urlParams) {
```

Comment:
Since this is an argument to `store`, not `webHelper` (which may not exist on some storage instances!), then I think it would be prudent to make it more generalized to helper config, rather than specifically to `webHelper`.  Maybe it should be called `opts` or `helperOpts` or something, and it could be passed to the `store` method of every helper. Each helper can decide whether those options are relevant to it or not. So for this use-case we could pass in `opts` as `{urlParams: <what we would have been passing in as urlParams>}`.